packages/ia-topnav/src/constants.js

```diff
@@ -0,0 +1 @@
+export const prodUrl = 'archive.org';
```

Comment:
SUGGESTION: Rather than it being a complete URL, I wonder if this constant is more clearly described as the "production hostname", since it's not really a complete URL. Also, I generally prefer not using abbreviations unless absolutely necessary or unless the abbreviation is *so* standard that there is zero cognitive load in understanding it. For me, "prod" is in the gray zone; I'd probably err on the side of being explicit.

So, I wonder if `productionHostname` or `productionHost` would be clearer for this constant.